mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age){
+  return "Hello, my name is " + name + " and I am " + age + " years old";
```

Comment:
That is a really great example of concatenation!  Another way to do this would be to use interpolation operator which can look like 

return `Hello, my name is ${name} and I am ${age}  years old`;

This is potentially a more readable and/or quicker way to insert code amongst a string.

Reply:
Awesome observation and suggestion. I do everything the long because I am able to see my work through, but it is such a messy end result. Thanks Mehreen...I'll try and remember this hun.